tests/suite_test.go

```diff
@@ -64,8 +64,10 @@ func TestMain(m *testing.M) {
 		log.Fatal(err)
 	}
 
-	defer teardownSuite(env)
-	os.Exit(m.Run())
+	exitCode := m.Run()
+	teardownSuite(env)
+
+	os.Exit(exitCode)
```

Comment:
I think the behaviour has changed: defer used to teardown the env even after a panic, but not now.

Reply:
Yes, but before `teardownSuite` was never executed as the process was terminated with `os.Exit`

Reply:
upd